AutomatedTesting/src/test/resources/features/RemovePlanet.feature

```diff
@@ -0,0 +1,10 @@
+Scenario outline: Users should be able to remove Planets from the Planetarium
```

Comment:
Feature Files start with "Feature: ..." on the first line and "Scenario Outline: ..." on the following line indented.